src/test/kotlin/com/igorwojda/integer/countdown/challenge.kt

```diff
@@ -4,7 +4,11 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countDown(n: Int): List<Int> {
-    TODO("not implemented")
+    val list = mutableListOf<Int>()
```

Comment:
Simplify this. Use ranges smarter and collectors/copying (https://kotlinlang.org/docs/reference/collection-aggregate.html). Can actually be an easy to read one-liner.